components/server/ee/src/prebuilds/prebuild-manager.ts

```diff
@@ -128,6 +128,11 @@ export class PrebuildManager {
             if (user.blocked) {
                 throw new Error(`Blocked users cannot start prebuilds (${user.name})`);
             }
+            if (!project) {
+                throw new Error(
```

Comment:
Can we throw a User Error (Invalid Argument) here?

Reply:
Good idea! I think this can be done. Let me try.